app/controllers/application_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ApplicationController < ActionController::Base
   layout :layout_by_resource
 
+  def check
+    render json: { status: "OK", version: ENV["SHA"], environment: Rails.env }, status: :ok
```

Comment:
where does ENV["SHA"] originate?

Reply:
It'll be set when the app is deployed. If it's not set, it'll be `nil`